app/main/views/frameworks.py

```diff
@@ -136,6 +137,33 @@ def label_columns(labels, query):
     })
 
 
+@main.route('/frameworks/<string:framework_slug>/suppliers', methods=['GET'])
+def get_framework_suppliers(framework_slug):
+    framework = Framework.query.filter(
+        Framework.slug == framework_slug
+    ).first_or_404()
+
+    agreement_returned = request.args.get('agreement_returned')
+
+    supplier_frameworks = SupplierFramework.query.filter(
+        SupplierFramework.framework_id == framework.id
+    )
+
+    if agreement_returned is not None:
+        if convert_to_boolean(agreement_returned):
+            supplier_frameworks = supplier_frameworks.filter(
+                SupplierFramework.agreement_returned == true()
```

Comment:
By having `== true()` here does it mean that both `?agreement_returned=true` and  `?agreement_returned=false` will return the list of suppliers who have returned their agreement, which seems odd.

Should it be `== convert_to_boolean(agreement_returned)` instead to get either suppliers who have or suppliers who haven't returned their agreements?  Or have I missed something?